multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/FlowableConfiguration.java

```diff
@@ -67,30 +70,29 @@ public SpringProcessEngineConfiguration processEngineConfiguration(DataSource da
         processEngineConfiguration.setAsyncExecutor(jobExecutor);
         // By default Flowable will retry failed jobs and we don't want that.
         processEngineConfiguration.setAsyncExecutorNumberOfRetries(0);
+        // In fact these configurations are for AsyncTaskExecutor
+        processEngineConfiguration.setAsyncExecutorThreadPoolQueueSize(configuration.getFlowableJobExecutorQueueCapacity());
+        processEngineConfiguration.setAsyncExecutorMaxPoolSize(configuration.getFlowableJobExecutorMaxThreads());
+        processEngineConfiguration.setAsyncExecutorCorePoolSize(configuration.getFlowableJobExecutorCoreThreads());
+        processEngineConfiguration.setAsyncExecutorSecondsToWaitOnShutdown(JOB_EXECUTOR_SHUTDOWN_AWAIT_TIME_IN_SECONDS);
```

Comment:
This is the `scale` method. The AsyncJobExecutor does not have setter for these as they are part of the AsyncTaskExecutor now.